ticker.py

```diff
@@ -0,0 +1,135 @@
+from time import time
+import json
+import toml
+import requests
+import argparse
+from urllib import urlencode
+
+PAGE_LIMIT = 200  # for aggregation endpoint
+
+
+def millis():
+    """current time in millis since epoch"""
+    return int(round(time() * 1000))
+
+
+def get_asset_params(prefix, atype, code, issuer):
+    """get aggregation request parameters for single asset"""
+    return {
+        prefix + "_asset_type": atype,
+        prefix + "_asset_code": code,
+        prefix + "_asset_issuer": issuer,
+    }
+
+
+def get_asset_param_from_pair(pair, prefix):
+    """get aggregation parameters for asset pair"""
+    if pair[prefix + "_asset_issuer"] == "native":
+        return get_asset_params(prefix, "native", "", "")
+    else:
```

Comment:
code style: this can be outside the else block. if we hit the first case we would return anyway and usually it's ok and sometimes recommended to have code that follows that outside of the if condition